src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/RecallControllerTest.kt

```diff
@@ -579,4 +611,188 @@ class RecallControllerTest {
 
     verify { recallService.confirmRecallType(recallId, request, currentUserId) }
   }
+
+  @Test
+  fun `missingDocuments is null when probationInfo not set`() {
```

Comment:
Scope and extent of tests here suggest that (for a future iteration?) we might have a `parameterizedTest` for this feature - perhaps via move the `missingDocuments` logic into a collaborator?

Reply:
Yep, I did think about this too. Would make sense at some point to extract